.github/workflows/ci.yml

```diff
@@ -76,6 +76,11 @@ jobs:
         run: |
           uncrustify --version
           find . -not -path '.*catch_assert.h' -iname "*.[hc]" -exec uncrustify --check -c tools/uncrustify.cfg {} +
+          if [ "$?" = "0" ]; then
+            exit 0
+          else
+            echo "To have the code uncrustified for you, please comment '/bot run uncrustify' on the Pull Request."
```

Comment:
Shouldn't this rather echo that the uncrustify job has failed? The `if` of the conditional checks for the uncrustify job status.

Reply:
Updated :)